requirements.txt

```diff
@@ -36,7 +37,6 @@ more-itertools==7.2.0
 nbconvert==5.5.0
 nbformat==4.4.0
 notebook==5.7.8
-numpy==1.16.4
```

Comment:
why you don't need numpy any more?

Reply:
> why you don't need numpy any more?

good you caught it. I uninstalled in my local and installed the latest version to solve some bugs and remove some warning but I forgot to update the requirements again. I will manage it.